src/irc/IdentGenerator.ts

```diff
@@ -27,6 +27,10 @@ export class IdentGenerator {
     private static readonly MAX_REAL_NAME_LENGTH = 48;
     // The max length of <username> in USER commands
     private static readonly MAX_USER_NAME_LENGTH = 10;
+    // The delimiter ofthe username.
+    private static readonly USER_NAME_DELIMITER = "_";
+    // The delimiter ofthe username.
+    private static readonly MAX_USER_NAME_SUFFIX = 10000;
```

Comment:
We talked a bit about this being weird to be set to 10000 instead of 9999.
If this was changed, the condition checking the length would need to become `<=` instead of `<`.